src/Controllers/AuthController.php

```diff
@@ -36,10 +36,12 @@ public function getLogin()
      */
     public function postLogin(Request $request)
     {
-        $credentials = $request->only(['username', 'password']);
+        $username = config('admin.database.users_username_field', 'username');
```

Comment:
this is here, because it's a new configuration option, so previous users might not have it